src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageServiceProtocol/LanguageServerTests.cs

```diff
@@ -343,6 +344,30 @@ public void TestDidOpenSeverityFormula()
             TestPublishDiagnostics("powerfx://app", "textDocument/didOpen", formula, expectedDiagnostics);
         }
 
+        [Fact]
+        public void TestConcatenateStartAndEndChars()
```

Comment:
>TestConcatenateStartAndEndChars [](http://example.com/codeflow?start=20&length=31)

Can we update to 
TestDidOpenWithErrors()

(it's not about concat). 

I'm seeing this is the only test that checks:
- in a case where the expression has errors. 
- on an /didOpen message, 
- Params.Diagnostics 
so we have a gap!!

Reply:
Done